src/EnergyPlus/CMakeLists.txt

```diff
@@ -727,8 +726,15 @@ endif()
 if (WIN32)
   target_link_libraries( energypluslib Shlwapi )
 endif()
+
+# we will build the intermediate wrapper DLL that hard links to Python DLL here
```

Comment:
Here is where we build the intermediate library.  Here's the build process:

- Build EnergyPlus static library which houses all the internal guts of EnergyPlus
- Build PythonWrapper which hard-links to the Python DLL and exposes functionality
- Build EnergyPlus shared library which does not hard link to the PythonWrapper, but rather delay-links to it at runtime, only if needed.  This shared library wrapper just takes the static lib and adds some API exposed functions.
- Build EnergyPlus executable which hard links to the EnergyPlus shared library.  This executable just contains basically a main function.
- If doing testing, the GTest unit test executable is built and linked to the EnergyPlus library -- which I assume to be the static library, though there seems to be stuff going on by CMake.

(If you are looking at this section as soon as I write this comment, you'll see I am actually linking the energypluslib to the pythonwrapper in CMake, which is counter to what I wrote above.  I am about to push a commit that changes the hard link and makes the python wrapper a simple build dependency for energyplusapi.)